x-pack/plugins/security_solution/public/common/components/event_details/__mocks__/index.ts

```diff
@@ -655,4 +655,16 @@ export const mockAlertDetailsData = [
     values: ['7.10.0'],
     originalValue: ['7.10.0'],
   },
+  {
+    category: 'threat',
+    field: 'threat.indicator',
+    values: [`{"verybad":true}`],
+    originalValue: [`{"verybad":true}`],
+  },
+  {
+    category: 'threat',
+    field: 'threat.indicator.matched',
+    values: [`{"verybad":true}`],
```

Comment:
Unless I'm misunderstanding the context here, It might be good to have different values for these two fields so that our assertions can be more explicit and we don't e.g. have a false positive if we accidentally use the same value in two places.